PC02_graffiti_BRAATEN.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep  4 15:09:16 2020
+
+@author: conno
+"""
+from turtle import *
+
+
+panel = Screen()
+w=750
+h=750
+panel.setup(width=w, height=h)
+
+Turtle()
+from turtle import *
+
+image = "Bezos.gif"
+panel.bgcolor("#ffa8cb")
+panel.bgpic(image)
+
+panel = Screen()
+
+t =Turtle()
+
+t.color("black","black")
+
+t.penup()
+t.left(90)
+t.forward(160)
+t.pendown()
+
+t.begin_fill()
+t.forward(40)

```

Comment:
Create a for loop to condense code creating a shape
```suggestion
for i in range(4):
t.forward(40) #indented
t.left(90) #indented

```